### ISSUE TITLE
Icons/buttons in web browser are too small

### ISSUE THREAD

**Issue by lapor** [ubports/ubports-touch#271]
- Device: M10 FHD
- Channel: RC
- Build: r6 
 
## Steps to reproduce

1. Open the web browser on the tablet M10 FHD

## Expected behavoir

Icons/buttons for close the tab or open the new tab should be bigger and I wouldn't expect to have a new tab icon between the settings of the web browser and the indicator panel.

## Actual behavior

I miss the x icon/button to close the tab, so I have to try again. Sometimes I hit the setting of web browser instead of new tab icon/button. 

## Logfiles and additional information

...

**Comment by jonnius** [comment]
Do you still have this issue with the new Browser Morph?

**Comment by lapor** [comment]
Yes, I think that Browser Morph still has the same UI issues as the previous one.

**Comment by cibersheep** [comment]
I agree that the problems with small icons comes when a lot of tabs are open. It needs a bit of practice but we might consider a different way of showing tabs? 
(Fun fact, the icons are of a similar size of staged browser but it feels smaller, tabs are higher though)

![imatge](https://user-images.githubusercontent.com/6640041/48975047-87c68700-f068-11e8-915d-5e1a9ee0f890.png)

![imatge](https://user-images.githubusercontent.com/6640041/48975064-f99ed080-f068-11e8-955c-20b4e83b72c5.png)

**Comment by Fuseteam** [comment]
is this still an issue?

**Comment by cibersheep** [comment]
The size of the icons on tablet is similar to previous ones
